### PR TITLE
[MNOE-603] Display custom and provisioning data json schema

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

File: src/app/views/orders/order.controller.coffee [hunk 1]
```diff
@@ -30,6 +36,16 @@
           vm.user = response.data.plain()
       )
 
+  vm.formatJson = (json) ->
```
Comment (alachaum) [inline src/app/views/orders/order.controller.coffee]: @hedudelgado @alexnoox @fgourichon Seriously? Don't we have better ways to generate a display from a json payload??

Comment (hedudelgado) [reply src/app/views/orders/order.controller.coffee]: The only other alternative I have in mind is creating a component that generates dynamically an HTML template depending on the JSON keys and values.

Comment (alachaum) [reply src/app/views/orders/order.controller.coffee]: Pretty much anything that does not involve string replace operations would be better.

So yes, generating HTML for a web page sounds like a good idea.

Comment (alexnoox) [reply src/app/views/orders/order.controller.coffee]: I addressed this issue in this PR: https://github.com/maestrano/mnoe-admin-panel/pull/113

**Pull request merged**